src/PdV.cpp

```diff
@@ -147,18 +139,11 @@ void PdV_kernel(
 //  @details Invokes the user specified kernel for the PdV update.
 void PdV(global_variables &globals, bool predict) {
 
-	double kernel_time;
+	double kernel_time = 0;
 	if (globals.profiler_on) kernel_time = timer();
 
 	globals.error_condition = 0;
 
-	int prdct;
```

Comment:
This looks important

Reply:
`prdct` is dead code I think, there were no further references to that and it's not inside a kernel or kernel capture